docs/eessi.md

```diff
@@ -0,0 +1,63 @@
+# EESSI Guide
+
+## How to load EESSI
+Loading an EESSI environment module:
+```[bash]
+source /cvmfs/software.eessi.io/versions/2023.06/init/lmod/bash
+```
+Activating EESSI environment:
+```[bash]
+source /cvmfs/software.eessi.io/versions/2023.06/init/bash
+```
+
+## GPU Support with EESI
+To enable GPU support you need a site-specific build that has `cuda` enabled. For guide to do this please refer to `docs/image-build`. This is
+because CUDA-drivers are host specific and EESSI can not ship NVIDIA drivers due to licensing + kernel specific constraints. This means that the
+host must provide the drivers in a known location (`host_injections`).
+
+### Using GPUs
+All CUDA-enabled software in EESSI expects CUDA drivers in a specific `host_injections` subdirectory.<br>
+```[bash]
+ls -l /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/amd/zen3/software/CUDA/12.1.1/bin/nvcc
+```
+The output of this should show a symlink to the EESSI `host_injections` dir like so:
+```[bash]
+lrwxrwxrwx 1 cvmfs cvmfs 109 May  6  2024 /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/amd/zen3/software/CUDA/12.1.1/bin/nvcc
+-> /cvmfs/software.eessi.io/host_injections/2023.06/software/linux/x86_64/amd/zen3/software/CUDA/12.1.1/bin/nvcc
+```
+To expose the Nvidia GPU drivers.
+```[bash]
+/cvmfs/software.eessi.io/versions/2023.06/scripts/gpu_support/nvidia/link_nvidia_host_libraries.sh
+```
+
+### Buidling with GPUs
+
+Checking `nvcc --version` and `which nvcc` to see if `CUDA` compiler is found.<br>
```

Comment:
```suggestion
Check `nvcc --version` and `which nvcc` to see if `CUDA` compiler is found.<br>
```

why's there  `<br>` tags in markdown? Just add an extra newline if you need a para.